src/wp-admin/includes/plugin.php

```diff
@@ -842,7 +842,7 @@ function deactivate_plugins( $plugins, $silent = false, $network_wide = null ) {
 	}
 
 	if ( $do_blog ) {
-		update_option( 'active_plugins', $current );
+		update_option( 'active_plugins', $current, false );
```

Comment:
Per https://github.com/WordPress/wordpress-develop/pull/6465#discussion_r1731880434, this needs to be reverted too.
```suggestion
		update_option( 'active_plugins', $current );
```